src/components/GraphFigure.tsx

```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+import type { ReactNode } from 'react';
+import CountUp from 'react-countup';
+import Link from 'next/link';
+import { useMemo } from 'react';
+
+export const GraphFigure = ({
+  percentage,
+  description,
+  link,
+  icon,
+  percentageClassName = 'text-primary',
+}: {
+  percentage: number;
+  description: string;
+  link: string;
+  icon: ReactNode;
+  percentageClassName?: string;
+}) => {
+  const calculPercent = useMemo(() => (value: number) => value * 100, [percentage]);
```

Comment:
je sais pas si j'aurais utilisé useMemo() ici et peut être pas comme ça.
S'il s'agit juste de faire percentage * 100, ça peut être dans une fonction utilitaire (et pas forcément dans le composant).
Si tu tiens absolument à ce que ce soit ici, peut être useCallback() serait plus approprié.

Reply:
Je rejoins Jordan sur l'utilitaire disons formatPercentage car c'est bien une fonction vouée à être réutilisable

Et sur le fait que faire un mémo pour une simple multiplication est un peu overkill

Par contre c'est pas du tout un useCallback.
useCallback ça porte bien son nom, c'est un callback, comme par exemple sur un onclick ou tu déclanches le callback suite au click justement

Le useMemo vraiment c'est un cache d'une valeur "expensive" en terme de processeur

Reply:
Ouais c'etait plus dans le sens ou 
´´´
useCallback(fn, deps) is equivalent to useMemo(() => fn, deps).
´´´
Donc si tu tiens vraiment a faire ca comme ca, ca a plus de sens de memoiser la fonction dans un callback qui est fait pour ca que dans un memo.

Apres je reconnais que de toutes facon pas grand chose justifie de faire ça comme ça ici.
Au contraire ca complexifie le code et ca bouffe de l'espace memoire react. Une bonne fonction a l'ancienne est encore ce qu'il y a de mieux !

Reply:
@jmunozz , @nigiraudo . fonction utilitaire crée